api/src/main/scala/hmda/api/model/InstitutionWrapper.scala

```diff
@@ -3,7 +3,7 @@ package hmda.api.model
 import hmda.model.institution.InstitutionStatus
 
 case class InstitutionWrapper(
-  id: Int,
+  id: String,
```

Comment:
I see how this fixes the problem of integrating with the UI, but the bigger problem imho is why the institution ID is an Int. For 2018 we have [LEI](http://www.sifma.org/issues/operations-and-technology/legal-entity-identifier/overview/) which is certainly a String of alphanumeric characters